management/server/policy.go

```diff
@@ -191,6 +191,16 @@ func (p *Policy) UpgradeAndFix() {
 	}
 }
 
+// AreAllRuleGroupsEmpty checks if all rule groups in the policy are effectively empty.
+func (p *Policy) AreAllRuleGroupsEmpty() bool {
+	for _, rule := range p.Rules {
+		if len(rule.Sources) != 0 && len(rule.Destinations) != 0 {
+			return false
+		}
+	}
+	return true
+}
+
```

Comment:
Having 0 groups in either source or destination should be impossible (only maybe now with network acl). Should we additionally check if any of the groups defined in src or dst have at least one peer?

Reply:
This makes sense. Checking whether the policy groups contain any peers will cover these two cases.